Gemfile

```diff
@@ -35,8 +35,12 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', '~> 11.1', platforms: %i[mri mingw x64_mingw]
   # Use rspec with extensions for unit testing
-  gem 'rspec', '~> 3.9', require: false
-  gem 'rspec-rails', '~> 4.0', require: false
+  gem 'rspec', '~> 3.9'
```

Comment:
Please re-add `require: false` unless it breaks something

Reply:
(Re-add it to `rspec-rails` too, it got cut off in my comment but it also missing it)

Reply:
Could only add `require: false` to `rspec-rails` without breaking `rails spec`.